OneOneOne/reciever/message_checks/check.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 from builder.pystache_message_builder import PystacheMessageBuilder
-from OneOneOne.definitions import TEMPLATE_PATH, XML_PATH
+from definitions import TEMPLATE_PATH, XML_PATH
```

Comment:
This change breaks the tests on my machine, it instead refers to the definitions file in the  nhais directory for some reason.